core/env/slice_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../../spec_helper'
+
+ruby_version_is "2.6" do
+  describe "ENV.slice" do
+    before :each do
+      @saved = {
+          "foo" => ENV["foo"],
+          "bar" => ENV["bar"],
+          "bat" => ENV["bat"],
+      }
+      @pairs = {
+          "foo" => "0",
+          "bar" => "1",
+          "bat" => "2",
+      }
+      ENV.update(@pairs)
+    end
+
+    after :each do
+      ENV.update(@saved)
+    end
```

Comment:
Could you make the before/after simpler, like I did in 377ea5ff3ad392f066627610f968e3e071b80cc3?
I'd like to not require having the rarely-used and unlikely-to-be-implemented-in-early-Ruby-implementations `ENV.update` just to run other ENV specs.
Also, `bat` doesn't seem needed.

Reply:
Sure, simpler.  I'd thought to still have multiples even after one is removed.

Reply:
Would ENV.merge! serve?  Or do you strongly prefer ENV[]?

Reply:
> Would ENV.merge! serve? Or do you strongly prefer ENV[]?

I prefer ENV[] in `before`/`after` to avoid relying on too many Hash-like methods for ENV which might not be implemented.